genestack_client/settings/genestack_config.py

```diff
@@ -100,13 +103,20 @@ def get_text(parent, tag):
                 host = get_text(user, 'host')
                 email = get_text(user, 'email')
                 password = get_text(user, 'password')
+                token = get_text(user, 'token')
                 if not password:
                     try:
                         import keyring
                         password = keyring.get_password(_PASSWORD_KEYRING, alias)
                     except Exception as e:
                         print e
-                self.add_user(User(email, alias=alias, host=host, password=password), save=False)
+                if not token:
+                    try:
+                        import keyring
+                        token = keyring.get_password(_TOKEN_KEYRING, alias)
+                    except Exception as e:
+                        print e
```

Comment:
I think we should handle absence of keyring module explicitly, don't simply print Python error message.

Reply:
Lets leave this out of this PR scope.